src/utils/add_new.py

```diff
@@ -1,14 +1,14 @@
-from os import system
+from os import system, path
+import config
 
 import getFaces
 
-
 CRED = '\33[91m'
 CGRN = '\33[42m'
 CEND = '\33[0m'
 
 if __name__ == '__main__':
-    path = '/lib/Auth/Facerec/roots/'
+    path = path.join(config.PATH, 'roots/')
```

Comment:
Gives `UnboundLocalError: local variable 'path' referenced before assignment` on this line. Let's keep it simple string concatenation for this PR (it will reduce testing efforts). And will use `pathlib` (`pathlib` is better when it comes to working with paths.) to replace those string paths with `pathlib.Path` in another PR as it's not in the scope of this PR.

Reply:
Okay then, for now I'll replace all the path.join with string concatenation